### PR TITLE
more discussion of x ≈ 0 in isapprox docs

### PR DESCRIPTION
**Pull request by stevengj**
This seems to be a FAQ.

### CODE REVIEW

File: base/floatfuncs.jl [hunk 1]
```diff
@@ -214,6 +214,17 @@ approximately equal component-wise.
 The binary operator `≈` is equivalent to `isapprox` with the default arguments, and `x ≉ y`
 is equivalent to `!isapprox(x,y)`.
 
+Note that comparing `x ≈ 0` (i.e., comparing to zero with the default tolderances)
```
Comment (helgee) [inline base/floatfuncs.jl]: s/tolderances/tolerances/

File: base/floatfuncs.jl [hunk 2]
```diff
@@ -214,6 +214,17 @@ approximately equal component-wise.
 The binary operator `≈` is equivalent to `isapprox` with the default arguments, and `x ≉ y`
 is equivalent to `!isapprox(x,y)`.
 
+Note that comparing `x ≈ 0` (i.e., comparing to zero with the default tolerances)
```
Comment (nalimilan) [inline base/floatfuncs.jl]: "comparing" -> "writing" (you don't compare `x ≈ 0` to anything).

BTW, I'm not sure it's common to say "you", "your" in the docs. I'd say imperative is more common, and "the problem (at hand)" is fine.

**Review by StefanKarpinski — APPROVED** [review]

File: base/floatfuncs.jl (hunk 2 above)
Comment (mvhulten) [reply base/floatfuncs.jl]: Neither "comparing" nor "writing" is correct.  "writing `x ≈ 0`" is not equivalent to "`x == 0`", it is not even equivalent to "writing `x == 0`".  Only `x ≈ 0` is equivalent to "`x == 0`".

**Review by mvhulten — CHANGES_REQUESTED** [review]
Even though the unit issue is clearly relevent, readers may think the reasoning is flawed as using cm for the Earth's diameter and km for the Bohr radius is atypical.  If both units are m, your reasoning still holds (giving a relative tolerance for comparing to the Earth diameter of `1.57e-15` which is very small, and one for comparing with the Bohr radius of `189` which is absurdly small.  (All in all, you could use `atol=1e-9`, giving an extra order of magnitude in the Earth radius case.)

Besides this detail, I think this is a good addition to the documentation!

**Comment by stevengj** [comment]
@mvhulten, I changed them both to meters.

**Review by mvhulten — APPROVED** [review]

**Review by nalimilan — APPROVED** [review]

**Pull request merged**